### ISSUE TITLE
Linter-gcc not working / not even displaying messages in console

### ISSUE THREAD

**Issue by bigboiblue** [hebaishi/linter-gcc#200]
## Checklist
- [x] Check that you have the GNU C Compiler installed by typing ```gcc --version``` into a terminal.
- [x] If you don't have the ```linter``` package installed, either update to the latest version of ```linter-gcc``` or install linter directly with ```apm install linter```.
- [x] Ensure that you are using a valid grammar - either C/C++/C++14
- [x] Ensure that you have read all the relevant information in the [wiki](https://github.com/hebaishi/linter-gcc/wiki), particularly the [Common Issues](https://github.com/hebaishi/linter-gcc/wiki/Common-Issues) page.

## Issue (including debugging messages):

Hey,
I have installed the latest versions of linter, linter-ui-default and linter-gcc. I am using:
```
Atom    : 1.34.0
Electron: 3.1.1
Chrome  : 66.0.3359.181
Node    : 10.2.0
```
I have also installed the latest gcc-compiler extension to enable me to compile c++ programs within atom. When I compile my C++ programs, which I have purposefully ridden with a bunch of errors, the linter shows no errors. The linter simply does nothing.

I have checked show debugging messages, and there are no messages regarding linter or linter-gcc in the developer console.

I do however repeatedly get this warning in the console:
`fs.js:958 (electron) The 'lstatSyncNoException' function has been deprecated and marked for removal.`
whenever I perform pretty much any action in atom, however, this may not be related at all, as this warning is displayed regardless of whether or not linter and linter-gcc is installed.

I would greatly appreciate it if you could help me out with this, as I find this to be a great package.

**Comment by mitchellgalea** [comment]
Did you end up figuring out what was wrong? I seem to be having the same issue

**Comment by Delran** [comment]
I suggest you switch to a previous version of Linter-gcc, at least until the release version is fixed,
There is a fork at this url, https://github.com/tvincent056/linter-gcc2, got pretty much all the current features but quite slower than the release.
